File: app/src/main/java/pl/mareklangiewicz/myblockssample/Fragment1.java
package pl.mareklangiewicz.myblockssample;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.view.LayoutInflater;
import android.view.MenuItem;
import android.view.View;
import android.view.ViewGroup;

import pl.mareklangiewicz.myfragments.MyFragment;
import pl.mareklangiewicz.myviews.IMyNavigation;

public class Fragment1 extends MyFragment {

    @Nullable @Override public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        inflateMenu(R.menu.local_menu1);
        return inflater.inflate(R.layout.fragment1, container, false);
    }

    @Override public boolean onItemSelected(IMyNavigation nav, MenuItem item) {
        log.w("[SNACK]Item: %s selected.", item.getTitle());
        return true;
    }
}

File: app/src/main/java/pl/mareklangiewicz/myintentsample/Fragment1.kt
package pl.mareklangiewicz.myintentsample

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

import pl.mareklangiewicz.myfragments.MyFragment
import pl.mareklangiewicz.myutils.*

class Fragment1 : MyFragment() {
    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        manager!!.lnav!!.menuId = R.menu.local_menu1
        manager!!.lnav!!.items { log.w("[SNACK]Item: $it selected.") } // TODO: manage subscription
        return inflater.inflate(R.layout.fragment1, container, false)
    }
}

